packages/storybook-utils/src/sbType.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { SBType } from "@storybook/csf";
```

Comment:
```suggestion
import type { SBType } from "storybook/internal/types";
```

You can import it like this so you can remove the `@storybook/csf` dependency